### ISSUE TITLE
[Question] It is possible to digitally sign an Excel workbook programmatically ?

### ISSUE THREAD

**Issue by Leafgard** [EPPlusSoftware/EPPlus#1300]
Hello,

I was wondering if it is possible to digitally sign an Excel workbook, as in [the following example](https://support.microsoft.com/en-gb/office/add-or-remove-a-digital-signature-for-microsoft-365-files-70d26dc9-be10-46f1-8efa-719c8b3f1a2d).

I have checked the docs, but didn't find out something else then VBA signing.

Thanks in advance !

**Comment by JanKallman** [comment]
No, this is currently not possible. I will add this issue as an enhancement.

**Comment by OssianEPPlus** [comment]
As of Epplus 8 [you can add digital signatures to a workbook](https://github.com/EPPlusSoftware/EPPlus/wiki/Digital-Signatures) (See link for wiki)
